boxsdk/object/base_api_json_object.py

```diff
@@ -61,3 +61,8 @@ def __repr__(self):
     def _description(self):
         """Return a description of the object if one exists."""
         return ""
+
+    @property
+    def response_object(self):
+        """Return the JSON object representing the object returned from a Box API request."""
```

Comment:
Document ``:rtype: `dict` ``.

Also, we might want to return a copy of the `dict`, that way if they make modifications to the dict it won't affect the smart object. I will think about which behavior is preferable..